### ISSUE TITLE
Changing the threshold for when scores can be dropped.

### ISSUE THREAD

**Issue by JMCanning78** [BlaiseRitchie/MahjongSite#1]
Hi Blaise.  This is John Canning.  I first wanted to confirm that I found the right GitHub repo for the Seattle Mahjongg club website.  Is this correct?

Assuming it is, we need to change the program to, at a minimum, go back to requiring at least 8 games before one score can be dropped from the average.  Right now, it's showing "6 (+1)" for some players in the 2nd Quarter.  Since we haven't had a snow day like we did in the 1st Quarter, we need to go back to what we had before.  On the other hand, at the beginning of the 2nd Quarter, I think the group voted to up the threshold for when players could drop 1 score from their average.  I think the new threshold was 9 or 10.  Edwin and Zach will probably know.

**Comment by JMCanning78** [comment]
Also, I have an interest in helping out with maintaining the code.  I just glanced at it overall, and saw the settings.py.example with the DROPGAMES threshold.  It wasn't clear to me where settings.py was (I assume the .example file is not used on the production web site).

What web server is used to run this?  Is it Tornado (http://www.tornadoweb.org/en/stable/)?  What database is used?  I'd like to set up a similar environment on my computer so I could test out contributions.

**Comment by JMCanning78** [comment]
I answered most of my questions above.  I was able to download the code and get it running.  It took me a little while to create an admin account from which to manage other users and settings, but I eventually got it working.  I plan to submit some pull requests with suggested additions and modifications.
The essence of this Github Issue, however, is just to change the DROPGAMES threshold in the settings.py file on the production web server.
